packages/dropdowns/src/elements/Multiselect/Multiselect.spec.tsx

```diff
@@ -530,19 +534,25 @@ describe('Multiselect', () => {
       const tags = getAllByTestId('tag');
       const input = container.querySelector('input');
 
-      fireEvent.focus(input!);
+      expect(tags[0]).not.toHaveFocus();
+
       fireEvent.keyDown(input!, { key: 'Home', keyCode: KEY_CODES.HOME });
 
       expect(tags[0]).toHaveFocus();
     });
 
     it('does not focus first tag on home keydown with input value', () => {
```

Comment:
This test was a false positive because the first `tag` did not have focus throughout the test lifecycle. I've added assertions to ensure that the first tag doesn't have focus prior to a user pressing "home" with the input value. Then I added assertions to ensure that the first tag does have focus after user interaction.